src/ast/nodes/MemberExpression.ts

```diff
@@ -445,8 +445,9 @@ export default class MemberExpression
 		destructuredInitPath: ObjectPath,
 		init: ExpressionEntity
 	): boolean {
+		let included = this.included;
```

Comment:
I was checking why we do not have any coverage for this method and of course, the reason is that `includeDestructuredIfNecessary` is currently only called for declarations, so this is basically dead code. As I do not trust untested code (in case we have run this logic also for assignments), I would suggest to "improve" coverage by replacing this method with a stub


```suggestion
		/* istanbul ignore next */
		this.scope.context.error(
			{
				message: 'includeDestructuredIfNecessary is currently not supported for MemberExpressions'
			},
			this.start
		);
```

This will throw an error with a frame so that users can give us helpful information if we overlook anything.

Reply:
That makes sense! Thanks for pointing out why this method isn’t executed. When I saw the coverage data, I was also curious about why this method wasn’t covered.